test/test_python_reproducibility.py

```diff
@@ -46,15 +46,15 @@ def __exit__(self, exc_type, exc_value, traceback):
         sys.stderr = self.old_stderr
 
 
-def WorkerInit():
+def worker_init():
     """Function run on initialization of a Pool worker process.
 
     If the setproctitle module is installed, this will adjust the process title (shown by ps) to be more informative.
     """
-    setproctitle('python worker %d' % test_support.GetProcessNumber())
+    setproctitle('python worker %d' % test_support.get_process_number())
 
 
-def RunExperiment(modelName, protoName, expectedOutputs):
+def run_experiment(modelName, protoName, expectedOutputs):
```

Comment:
Haven't changed arg names. But this isn't testable at present so probably best not to touch this file yet?